All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+## 7. Using the Projects feature of Github to organize talking points for future EIPIP meetings.
+
+**Pooja Ranjan**
+So yeah, we can start with the idea of the project board. That is something new and interesting that we might need brainstorming over there. So just introduce why and how you thought of it and where you see this going.
+
+**James Hancock**
+Okay. You want to talk about your thoughts on this?
+
+**Edson Ayllon**
+Yeah, so I was watching you use it for EFI. And it looks like Trello. Sometimes, there's things on the agenda, which we don't get to. As more and more items are added to the agenda, I think it'll add bloat to future agendas, and we might lose items. So I think it's a good way to organize future talking points that we table to future meetings. Or sometimes they have agenda items that - say we want to focus on decision making, but we're also introducing stuff around on-boarding. We still have ideas that we want to keep, but not necessarily address at this moment. And a way to organize them could be using the project's feature and GitHub. So it's more of a way to organize the topics from these meetings.
+
+**Pooja Ranjan**
+Yeah, I kind of like the idea, too. Sometimes back, we have used it for creating the post-mortem for constant in poll and the Cat Herders itself. I'm gonna share the link in the chat here. So yeah, this is a good way to keep track of whatever agenda that we are trying to discuss here, and of any decisions we make, and how we are going to implement that. It would be a good way to keep track of it.
+
+**James Hancock**
+Yeah, this has the Cat Herders EIP repo use project boards before? Has it set up for them?
+
+**Pooja Ranjan**
+Yeah, when we look into the EIPIP repo, there is a section below project which is empty. At this point of time, I'm going to share the link here again. So there we can kind of sort tasks, plan project, automate workflow and everything is there. So we might have to take what we want to put where, or how we would want to address this.
+
+**James Hancock**
+Great. I'd love to see us move forward on this, or at least investigate where to get integrated and such.
+
+**Pooja Ranjan**
+So I think up 'til now, how we have been managing is we had this initial document of EIPIP where we gathered some idea how we are going to do that. And the extensive report that came up after the EIPIP survey. So that was another reference point. And now we have to collect the main ideas from there and add them as tasks. And then we can keep measuring the progress. We need to keep track of the progress. And if it is closed, we can certainly reach it out to everybody in community as well as the CoreDevs to let them know what the things are.
```

Comment:
So I think up 'til now, how we have been managing as we had this [initial document](https://docs.google.com/document/d/1KzEMD21Joqn7IWk4WnocwraRuo66y7r2w8R9Wml4BN0/edit) of EIPIP where we gathered some idea how we are going to do that. And the extensive report that came up after the EIPIP survey. So that was another reference point. And now we have to collect the main ideas from there and add them as tasks. And then we can keep measuring the progress. We need to keep track of the progress. When it is closed, we can let everybody in the community as well as the CoreDevs know what the changes are.